### PR TITLE
cli: correct the error message when receiving http errors.

### PR DESCRIPTION
**Pull request by yorkie**
To reproduce the bug that this PR tries to fix, just run:

```sh
$ pipcook run https://foobar
```

### CODE REVIEW

**Comment by yorkie** [comment]
And this fixes #337.

File: packages/cli/src/request.ts [hunk 1]
```diff
@@ -51,12 +55,18 @@ export const listen = async (host: string, params?: RequestParams, handlers?: Re
...
         es.close();
         clearTimeout(timeoutHandle);
-        console.error('daemon is not started, run "pipcook daemon start"');
         es.removeEventListener('error', onerror);
+
+        // print error
+        if (typeof e?.status === 'number') {
+          console.error('occurrs an daemon error with the following response:\n', e);
+        } else {
+          console.error(`daemon is not started(${e.message}), run "pipcook daemon start".`);
```
Comment (FeelyChau) [inline packages/cli/src/request.ts:68]: if `e` is undefined, throw an error here.

Comment (yorkie) [reply packages/cli/src/request.ts:68]: See https://github.com/EventSource/eventsource/blob/master/lib/eventsource.js#L53, https://github.com/EventSource/eventsource/blob/master/lib/eventsource.js#L143 and https://github.com/EventSource/eventsource/blob/master/lib/eventsource.js#L162, the upstream library `eventsource` guarantee this won't be a undefined value.

Comment (FeelyChau) [reply packages/cli/src/request.ts:68]: If `e` is not undefined, why we check it by `if (typeof e?.status === 'number') {`?

**Comment by yorkie** [comment]
Updated, thank you @FeelyChau :)

**Pull request merged**